roles/pulp_webserver/meta/main.yml

```diff
@@ -9,7 +9,7 @@ galaxy_info:
   platforms:
```

Comment:
(Not a comment against this line, but I do not know how to reference an unmodified file.)

roles/pulp_devel/tasks/install_podman.yml has workarounds to put newer (backported) podman on Debian 10. We can clean that up now.